### PR TITLE
Add change to test trigger output

### PR DESCRIPTION
**Pull request by m0nggh**
Used OGP ds to display the info box instead:

image

### CODE REVIEW

File: packages/frontend/src/components/TestSubstep/index.tsx [hunk 1]
```diff
@@ -163,11 +165,22 @@ function TestSubstep(props: TestSubstepProps): React.ReactElement {
 
           {stepsWithVariables.length === 1 && (
             <Box sx={{ width: '100%' }}>
-              <Alert severity="info" sx={{ width: '100%' }}>
-                We found these variables:
-              </Alert>
+              <Infobox
+                icon={<BiCheck style={{ color: '#0F796F' }} />}
```
Comment (ogp-weeloong) [inline packages/frontend/src/components/TestSubstep/index.tsx]: nit: can directly use `color` prop.

Let's move this to a chakra token in a later PR.

File: packages/frontend/src/components/TestSubstep/index.tsx [hunk 2]
```diff
@@ -163,11 +165,22 @@ function TestSubstep(props: TestSubstepProps): React.ReactElement {
...
+                  background: '#E2EEED',
+                }}
+              >
+                Here is the test data we found. You can use these as variables
+                in your action steps below.
+              </Infobox>
               <Box
-                sx={{ maxHeight: 400, overflowY: 'auto', width: '100%' }}
+                sx={{
+                  maxHeight: 400,
```
Comment (ogp-weeloong) [inline packages/frontend/src/components/TestSubstep/index.tsx]: Let's use rem for this: `25rem`

**Review by ogp-weeloong — APPROVED** [review]
LGTM

File: packages/frontend/src/components/TestSubstep/index.tsx (hunk 1 above)
Comment (m0nggh) [reply packages/frontend/src/components/TestSubstep/index.tsx]: okk

File: packages/frontend/src/components/TestSubstep/index.tsx (hunk 2 above)
Comment (m0nggh) [reply packages/frontend/src/components/TestSubstep/index.tsx]: donezo

**Pull request merged**